src/views/Nft/market/hooks/useGetLowestPrice.ts

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react'
+import { getNftsMarketData } from 'state/nftMarket/helpers'
+import { NftToken } from 'state/nftMarket/types'
+import { pancakeBunniesAddress } from '../constants'
+
+export interface LowestNftPrice {
+  isFetching: boolean
+  lowestPrice: number
+}
+
+export const getBunnyIdFromNft = (nft: NftToken): string =>
+  nft.attributes?.find((attr) => attr.traitType === 'bunnyId').value.toString()
```

Comment:
Can you add guards here for nft without bunny id?
```js
const bunnyId = nft.attributes?.find((attr) => attr.traitType === 'bunnyId')?.value
return bunnyId ? bunnyId.toString() : null
```

Just realized this would fail for other collections

Reply:
Sure - makes sense to add those catches.

This shouldn't be used for collections without a `bunnyID` traitType.

Reply:
yeah but if you'd refactor the hook for my suggestion about it might be that other collection calls this function. Result will be ignored in this case though. :)

Reply:
Yea sorry I didn't realise these suggestions came as a package! Makes total sense in the context of the other.

I liked this catch regardless tho so added it 😉